docs/designs/2686-proposal-process.md

```diff
@@ -0,0 +1,96 @@
+# Proposing Changes to Tanzu Community Edition
+
+* Proposal: [https://github.com/vmware-tanzu/community-edition/issues/2686](https://github.com/vmware-tanzu/community-edition/issues/2686)
+
+## Introduction
+
+Tanzu Community Edition relies on design proposals to assess the viability of non-trivial changes
+to the project. This document covers details on the proposal process, which can also be used as
+guidelines for how to create a proposal.
+
+## The Proposal Process
+
+The proposal process is the process for reviewing a proposal and reaching a decision about whether
+to accept or decline the proposal.
+
+1. The proposal author [creates a brief issue](https://golang.org/issue/new) describing the proposal.
+
+   > A design doc is **not** required at this point; although proposal authors
+   > are welcome to create one.
+
```

Comment:
Apparently, MD lint doesn't like a blank line here inside the block quote:
```
docs/designs/2686-proposal-process.md:20 MD028/no-blanks-blockquote Blank line inside blockquote
```

Question: do we want to enable MD linter inside of the `docs/designs/` directory or should we disable it, giving the author the freedom to format it how they will?

Reply:
I do think it's useful having the mdlinting to make sure the markdown is well formatted and will render properly.

Reply:
I agree.

I added the following to the design doc section:

```
_note: PRs for design docs are linted using
[markdownlint](https://github.com/markdownlint/markdownlint), you can run it
locally or use `make check`._
```